### DISCUSSION TITLE
Documentation clarification about knative 1.0 and kserve 0.8

### DISCUSSION THREAD

**Discussion by elukey** [kubeflow/kfserving#2303] [General]
/kind bug

Hi folks, I am opening this issue to get a clarification about kserve 0.8's requirements, especially related to knative. As far as I can see from the release's [changelog](https://kserve.github.io/website/0.8/blog/articles/2022-02-18-KServe-0.8-release/), Knative 1.0 seems now supported and usable. At Wikimedia we are looking for upgrading from 0.7 to 0.8, but we still have Knative 0.18 (since our Kubernetes cluster is running version 1.16, we are looking into upgrading to 1.23 but it will take a bit since it is a self-hosted/managed deployment).

When I looked at https://github.com/kserve/kserve/pull/1969 I noticed a lot of API changes for Knative 1.0, and at first sight it seems as if older versions are not supported anymore. Could you clarify if kserve 0.8 can be deployed with older versions of Knative or if 1.0 is the minimum requirement going forward?

Thanks a lot!

**Comment by yuzisun** [comment]
@elukey Knative 1.0 is definitely recommended minimum version for KServe 0.8, though the user facing Knative API was already v1 with 0.18 and the changes you noticed in that PR are mostly for Knative client go library used in KServe, in theory KServe 0.8 can still work Knative 0.18 but certainly KServe team has not done extensive testing with 0.18 version and you probably need to test out this combination if you really want to go with this route.

**Reply by elukey** [reply]
Thanks a lot makes sense!
